app/scripts/components/datasets/s-usage/index.js

```diff
@@ -43,7 +43,7 @@ function DatasetsUsage() {
         <FoldProse>
           {datasetUsage?.url && datasetUsage?.title ? (
             <p>
-              Check out how to use this dataset in this example notebook:{' '}
+              Check out how to use this dataset in this example notebook (using <code>collection='{dataset.data.layers[0].id}'</code>):{' '}
```

Comment:
Specifying the layer index is not ideal obviously; is there an easy way to create a list of the layer ids? The only alternative I can think of is explicitly calling out a layer id (either as a string or as an index) in the usage object. But that seems not very "dry" if you are repeating either the string of the layer id or the layer index, since both of those references could change and the developer wouldn't realize the change in the usage reference.

Reply:
@abarciauskas-bgse I'm not sure what the goal is.
We can easily display a list of layers on that page. Is each one going to get a different notebook url? Is the layer id a parameter to the url?

Reply:
The goal is to list all the STAC collection ids for the current dataset. When you have the stac collection ID you can modify the example notebooks easily to explore your dataset of interest. For example in https://nasa-impact.github.io/veda-documentation/timeseries-rioxarray-stackstac.html you'll see there is a line:

```
collection = 'no2-monthly'
```

This is the first id in the list of layers for this dataset https://github.com/NASA-IMPACT/delta-config/edit/develop/datasets/no2.data.mdx, however the user might also be interested in the second layer `no2-monthly-diff` so if we could create a list that would be ideal. Alternatively we can point the user to how to list STAC collection IDs (via STAC API or this notebook https://nasa-impact.github.io/veda-documentation/list-collections.html) but I like the ability to be able to hand the user the collection IDs they might be interested in if they have landed on this usage page.